docs/modules/pe.rst

```diff
@@ -924,6 +924,10 @@ Reference
 
         Data after being decrypted by XORing it with the key.
 
+    .. c:member:: hash
```

Comment:
I would name this `md5` instead of `hash`, as it conveys more information about the kind of hash without having to go to the documentation.

Reply:
Is there a reason we don't want to use "hash.md5()" here?

Reply:
Fair enough, as we already have the offset and size for the rich header using `hash.md5()` should be enough. And as bonus you can use the hashing algorithm you want.

Reply:
Yes, md5 sounds good. Regarding using a hash.md5(), if I understand the function correctly, it operates above a specified chunk of the file, but the rich header hash is done above the decrypted rich data with the rolling xor key, in the file, it would still be encrypted.

Reply:
Oh, I see I can also pass the clear data directly to the hash module. 
`hash.md5(pe.rich_signature.clear_data) == "..."`
Cool, I guess hash.md5() is enough then.